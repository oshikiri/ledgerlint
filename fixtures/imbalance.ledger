2020-03-26 * toilet paper
  Expences:Household essentials  200 JPY
  Assets:Cash  -2000 JPY
