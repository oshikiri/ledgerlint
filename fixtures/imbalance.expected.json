{"file_path":"fixtures/imbalance.ledger","line_number":1,"level":"ERROR","error_message":"imbalanced transaction, (total amount) = -1800 JPY"}
